### DISCUSSION TITLE
Milvus Benchmarking using VectorDBBench

### DISCUSSION THREAD

**Discussion by navneetrajput06** [milvus-io/milvus#27478] [Q&A and General discussion]
Hi team,

i am using below benchmarking tool but i am wondering how to use URI for my milvus database.
We do have https enabled and that tool is not having any option to use SSL certificate.
https://github.com/zilliztech/VectorDBBench/blob/main/README.md#install-requirements

**Comment by filip-halt** [comment]
Im not sure it is compatible at the moment. @liliu-z
